docs/src/pages/[platform]/getting-started/accessibility/accessibility.angular.mdx

```diff
@@ -0,0 +1,5 @@
+import { Fragment } from '@/components/Fragment';
+
+<Fragment>
+  {({ platform }) => import(`./intro.mdx`)}
```

Comment:
Thoughts on adding more content for Vue/Angular? We don't have the primitives content to link to (and Authenticator doesn't currently have any accessibility specific content to link to I don't think)

Reply:
Yea, I think it should be good for now